site2/docs/reference-configuration.md

```diff
@@ -110,6 +110,7 @@ BookKeeper is a replicated log storage system that Pulsar uses for durable stora
 |readBufferSizeBytes|The number of bytes we should use as capacity for BufferedReadChannel.|4096|
 |writeBufferSizeBytes|The number of bytes used as capacity for the write buffer|65536|
 |useHostNameAsBookieID|Whether the bookie should use its hostname to register with the coordination service (e.g.: zookeeper service). When false, bookie will use its ip address for the registration.|false|
+|bookieId | If you want to custom bookie ID or use a dynamic network address for the bookie, you can set `bookieId`. <br><br>Bookie advertises itself using `bookieId` rather than `BookieSocketAddress` (`hostname:port` or `IP:port`). If you set `bookieId`, then `useHostNameAsBookieID` does not take effect.<br><br>`bookieId` is a non empty string that can contain ASCII digits and letters ([a-zA-Z9-0]), colons, dashes, and dots. <br><br>For more information about `bookieId`, see [here](http://bookkeeper.apache.org/bps/BP-41-bookieid/).|N/A|
```

Comment:
```suggestion
|bookieId | If you want to custom a bookie ID or use a dynamic network address for the bookie, you can set `bookieId`. <br><br>Bookie advertises itself using `bookieId` rather than `BookieSocketAddress` (`hostname:port` or `IP:port`). If you set `bookieId`, then `useHostNameAsBookieID` does not take effect.<br><br>`bookieId` is a non-empty string that can contain ASCII digits and letters ([a-zA-Z9-0]), colons, dashes, and dots. <br><br>For more information about `bookieId`, see [here](http://bookkeeper.apache.org/bps/BP-41-bookieid/).|N/A|
```